bodybank/character_generator/models.py

```diff
@@ -0,0 +1,46 @@
+from django.db import models
+
+class Ego(models.Model):
+    ego_name = models.CharField(max_length=50)
+    ego_cog_apt = models.IntegerField("Cognition (COG)")
+    ego_int_apt = models.IntegerField("Intuition (INT)")
+    ego_ref_apt = models.IntegerField("Reflex (REF)")
+    ego_sav_apt = models.IntegerField("Savy (SAV)")
+    ego_som_apt = models.IntegerField("Somatics (SOM)")
```

Comment:
"ego" is already part of the model name, what if we call the columns full words (cognition, intuition, reflex, etc), maybe more readable?